tsl/src/nodes/decompress_chunk/vector_quals.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file and its contents are licensed under the Timescale License.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-TIMESCALE for a copy of the license.
+ */
+#pragma once
+
+#include <postgres.h>
+#include <executor/tuptable.h>
+#include <nodes/primnodes.h>
+
+#include "vector_predicates.h"
+#include <compression/arrow_c_data_interface.h>
```

Comment:
Nit: I usually reserve `<>` includes for third-party libraries and use `""`-includes for anything that reference code that is being built.

```suggestion
#include "compression/arrow_c_data_interface.h"
```